src/core/linux/SDL_evdev_capabilities.h

```diff
@@ -35,10 +35,11 @@
 #define EVDEV_LONG(x)        ((x) / BITS_PER_LONG)
 #define test_bit(bit, array) ((array[EVDEV_LONG(bit)] >> EVDEV_OFF(bit)) & 1)
 
-extern int SDL_EVDEV_GuessDeviceClass(const unsigned long bitmask_ev[NBITS(EV_MAX)],
-                                      const unsigned long bitmask_abs[NBITS(ABS_MAX)],
-                                      const unsigned long bitmask_key[NBITS(KEY_MAX)],
-                                      const unsigned long bitmask_rel[NBITS(REL_MAX)]);
+extern SDL_UDEV_deviceclass
+SDL_EVDEV_GuessDeviceClass(const unsigned long bitmask_ev[NBITS(EV_MAX)],
```

Comment:
SDL is generally consistent about not putting a newline after the return type.

Even in coding styles where there is a newline after the return type, it is only done for definitions (implementations), not for declarations:

```
/* foo.h */
int foo(int bar, const char *baz);

/* foo.c */
int
foo(int bar, const char *baz)
{
    printf("called foo(%d, %s)\n", bar, baz);
}
```

The advantage of this is that when you grep for `^SDL_EVDEV_GuessDeviceClass\>` you only find the definition, and not declarations or calls.

Reply:
Thanks! Done.